frontend/src/container/ListOfDashboard/DashboardList.styles.scss

```diff
@@ -64,7 +64,6 @@
 
 						.dashboard-icon {
 							display: inline-block;
-							margin-top: 4px;
 							margin-right: 4px;
```

Comment:
@SagarRajput-7 instead of adding margin right add gap for the parent class (wrapper). In this case `title-link`